schemas/core/components/configurator.json

```diff
@@ -96,26 +96,13 @@
       },
       "required": ["id", "name", "default"]
     },
-    "customerSelection": {
-      "type": "object",
-      "patternProperties": {
-        "^[a-zA-Z0-9\\-]+$": {
```

Comment:
Flatten this out, because we have not run into situation that we receive any unregistered properties